fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2434,6 +2434,9 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int lake_compaction_fail_history_size = 12;
 
+    @ConfField(mutable = true)
```

Comment:
worry about this misconfiguration, although usually it is not expected to touch this configure in OSS version. But it will take effect if user is trying change it.

Reply:
Yes, but an error will be reported